pkg/driver/driver_test.go

```diff
@@ -0,0 +1,56 @@
+package driver_test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/scality/mountpoint-s3-csi-driver/pkg/driver/node/envprovider"
+)
+
+// validateEndpointURL is a function that mimics the validation in driver.NewDriver
+// but can be tested without all the dependencies of the full driver
```

Comment:
I think we don't need the unit tests that use `validateEndpointURL` since the next tests check the actual code that makes the endpoint url mandatory.

Reply:
I want to keep it, I plan to later expand it :)

Reply:
And you are right with the current context.